data-plane/dispatcher/src/test/java/dev/knative/eventing/kafka/broker/dispatcher/impl/filter/subscriptionsapi/CeSqlFilterTest.java

```diff
@@ -51,7 +51,7 @@ static Stream<Arguments> testCases() {
                 Arguments.of(event, "'TRUE'", true),
                 Arguments.of(event, "'FALSE'", false),
                 Arguments.of(event, "0", false),
-                Arguments.of(event, "1", false),
+                Arguments.of(event, "1", true),
```

Comment:
wat?

Reply:
why was that false before? :sweat_smile:

Reply:
Oh before integer to boolean casting was a cast error, which turns in a filter result of false. In v1, we defined integer to boolean casting so now it works :)